### DISCUSSION TITLE
blog/2024/11/29/test-in-contest/

### DISCUSSION THREAD

**Discussion by giscus** [ZnPdCo/blog#6] [Announcements]
# blog/2024/11/29/test-in-contest/

https://znpdco.github.io/blog/2024/11/29/test-in-contest/

**Comment by tangninghaha** [comment]
这个测空间的不一定准吧。

参见：https://www.cnblogs.com/caijianhong/p/18502933

**Reply by ZnPdCo** [reply]
谢谢补充。

这是真的，但是不需要这么准确

考场上我一般直接st-ed和这个应该就可以了

如果很接近上限还是要注意一下呀
